### PR TITLE
Updating index.md object specifications

### PR DESCRIPTION
**Pull request by shiralitech**
Changing 'Info' to 'UID'

thoughts on including/changing link to direct readers to: https://github.com/MystenLabs/sui/pull/3624 (more updates have occurred since original July 15th link attached in index.md "See what's new" section)

### CODE REVIEW

**Comment by Clay-Mysten** [comment]
> Changing 'Info' to 'UID'
> 
> thoughts on including/changing link to direct readers to: #3624 (more updates have occurred since original July 15th link attached in index.md "See what's new" section)

Thanks so much for the change, @shiralitech!

And good point. I updated the links in a second commit here. Asking @tnowacki to review this change. We may even move this line item back up toward the top of the list for better visibility. Todd, what do you think?

File: doc/src/learn/index.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ The following list includes the recent updates to Sui and the documentation:
...
 * Sui now supports development using [Microsoft Windows 11, macOS, and Linux](../build/install.md#supported-oses). See [install Sui](../build/install.md#prerequisites) for the prerequisites of each operating system.
 * This site is now available in two versions in the menu at top left: the default and stable [Devnet](https://docs.sui.io/devnet/learn) branch and the [Latest build](https://docs.sui.io/learn) upstream `main` branch. Use the `devnet` version for app development on top of Sui. Use the Latest build `main` branch for  [contributing to the Sui blockchain](../contribute/index.md) itself. Always check and submit fixes to the `main` branch.
-* `sui::id` is now `sui::object` and `VersionedID` is now `Info`. Use the [Object module](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/sources/object.move) that has [replaced the former ID.move.](https://github.com/MystenLabs/sui/pull/3241)
+* `sui::id` is now `sui::object` and `Info` is now `UID`. Use the [Object module](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/sources/object.move) that has [replaced the former ID.move](https://github.com/MystenLabs/sui/pull/3624).
```
Comment (tnowacki) [inline doc/src/learn/index.md:23]:
```suggestion
* `sui::id` is now `sui::object` and `Info` is now `UID` (as of [#3624](https://github.com/MystenLabs/sui/pull/3624)). Use the [Object module](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/sources/object.move) that has [replaced the former ID.move](https://github.com/MystenLabs/sui/pull/3241).
```

**Review by Clay-Mysten — APPROVED** [review]
Thanks again. Let's make this change.

File: doc/src/learn/index.md (hunk 1 above)
Comment (tnowacki) [reply doc/src/learn/index.md:23]: Thoughts on this? The existing change is a bit weird as the second PR did not rename ID to object

**Pull request merged**